Utils/azuretools-core/pom.xml

```diff
@@ -260,6 +260,13 @@
             <groupId>com.microsoft.azure.appplatform.v2019_05_01_preview</groupId>
             <artifactId>azure-mgmt-appplatform</artifactId>
         </dependency>
+	<dependency>
+        <!--<groupId>com.microsoft.azure.mysql.v2020_01_01</groupId>-->
+        <groupId>com.microsoft.azure.mysql.v2020_01_01</groupId>
+		<artifactId>azure-mgmt-mysql</artifactId>
+        <!--<version>1.0.0-beta-4</version>-->
```

Comment:
remove comments like this,  what's the purpose to leave this as comment?

Reply:
it will be updated after mysql sdk releases.